Missing 0 return value in copy_sasl_output

This prevents a compilation failure with current compilers.

Upstream: <http://lists.askja.de/pipermail/ldapvi/2023-September/000134.html>

diff --git a/ldapvi.c b/ldapvi.c
index 91b18bb30814ca81..89d231079881313d 100644
--- a/ldapvi.c
+++ b/ldapvi.c
@@ -1465,7 +1465,7 @@ copy_sasl_output(FILE *out, char *sasl)
 	int line = 0;
 	int c;
 
-	if (lstat(sasl, &st) == -1) return;
+	if (lstat(sasl, &st) == -1) return 0;
 	if ( !(in = fopen(sasl, "r"))) syserr();
 
 	if (st.st_size > 0) {
